examples/hello-world.yaml

```diff
@@ -8,5 +8,5 @@ spec:
   - name: whalesay
     container:
       image: docker/whalesay:latest
-      command: [cowsay]
-      args: ["hello world"]
+      command: [sh, -c]
+      args: ["sleep 30"]
```

Comment:
Why is this needed?

Reply:
This is a mistake. Thanks for catching this.

Reply:
Fixed.